pkg/apis/apps/v1beta1/register.go

```diff
@@ -35,7 +37,7 @@ func Resource(resource string) schema.GroupResource {
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, addDefaultingFuncs, addConversionFuncs)
-	AddToScheme   = SchemeBuilder.AddToScheme
+	AddToScheme   = announced.CombineSchemeBuilders(SchemeBuilder.AddToScheme, v1.AddToScheme).AddToScheme
```

Comment:
> please comment why. Which types are needed?

It includes at least a pod spec, which ripples out.  It's neat.  We die on duplicate registration of identical types, so this is hosed for 1.6.